frontend/src/app/pages/export-settings/export-settings.component.html

```diff
@@ -96,7 +113,7 @@ <h2>Marschzeittabellen erstellen</h2>
 
           <div class="slider">
             <label>Kartenmassstab: 1:{{ slider.value }}</label>
-            <mat-slider #scale min="10000" max="100000" step="5000">
+            <mat-slider #scale min="5000" max="100000" step="5000">
```

Comment:
können wir da einen default setzen? (z.b. 10'000 oder sogar 15'000).

Reply:
default is set here: https://github.com/cevi/automatic_walk-time_tables/blob/52290cbdde6f7c073987258c6e07a1829541e8c4/frontend/src/app/pages/export-settings/export-settings.component.ts#L32

I changed the default to 1:15'000